### PR TITLE
Fixes #16128: We should not start agent daemons outside of the service

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/16128

### CODE REVIEW

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

File: share/commands/server-disable-policy-distribution [hunk 1]
```diff
@@ -8,13 +8,29 @@
 # First, prevent the agent from restarting cf-serverd
 touch /opt/rudder/etc/disable-policy-distribution
 
-# Then, actually stop the server right now
-if type killall > /dev/null 2> /dev/null; then
-  killall cf-serverd
-elif type pidof > /dev/null 2> /dev/null; then
-  kill $(pidof cf-serverd 2>/dev/null)
+if type systemctl >/dev/null 2>&1; then
```
Comment (peckpeck) [inline share/commands/server-disable-policy-distribution]: Isn't it always the case on server distro ?

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
OK, squash merging this PR

**Pull request merged**